lib/srv/db/postgres/sql/redshift-activate-user.sql

```diff
@@ -9,7 +9,11 @@ BEGIN
 
     -- If the user already exists and was provisioned by Teleport, reactivate
     -- it, otherwise provision a new one.
-    IF EXISTS (SELECT user_id FROM svv_user_grants WHERE user_name = username AND admin_option = false AND role_name = 'teleport-auto-user') THEN
+    IF EXISTS (
```

Comment:
Are these reformattings necessary for the PR?

Reply:
no, I can remove them